modules/apps/object/object-web/src/main/resources/META-INF/resources/js/ObjectDefinitionsFDSPropsTransformer.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function propsTransformer({...otherProps}) {
+	return {
+		...otherProps,
+		onActionDropdownItemClick({
+			action,
+			itemData,
+		}: {
+			action: any;
+			itemData: any;
```

Comment:
Avoid using the `any` type, be as strict as possible.

If you have no idea about the type to use, you better start using unknown type instead:
```suggestion
			action: unknown;
			itemData: unknown;
```